source/process/marketing-guidelines.rst

```diff
@@ -1,5 +1,5 @@
 ============================================================
-Mattermost Marketing Content Guidelines
+Mattermost Community Content Guidelines
```

Comment:
Not sure I agree with changing this to "community" guidelines? The entire doc except the last new section is about marketing.